tests/test_standard_tags.py

```diff
@@ -139,18 +140,18 @@ def test_multiple_matching_substring_legacy(self):
 
     def setup_multiple_matching_substring(self):
         self.request_multiple_matching_substring = weblog.get(
-            "/waf?token=03cb9f67dbbc4cb8b9&key1=val1&key2=val2&pass=03cb9f67-dbbc-4cb8-b966-329951e10934&public_key=MDNjYjlmNjctZGJiYy00Y2I4LWI5NjYtMzI5OTUxZTEwOTM0&key3=val3&json=%7B%20%22sign%22%3A%20%22%7D%7D%22%7D&ecdsa-1-1%20aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa=%09test"  # pylint: disable=line-too-long
+            "/waf?token=03cb9f67dbbc4cb8b9&key1=val1&key2=val2&pass=03cb9f67-dbbc-4cb8-b966-329951e10934&public_key=MDNjYjlmNjctZGJiYy00Y2I4LWI5NjYtMzI5OTUxZTEwOTM0&key3=val3&application-key=dogkey&json=%7B%20%22sign%22%3A%20%22%7D%7D%22%7D&ecdsa-1-1%20aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa=%09test&json=%7B%20%22app-key%22%3A%20%22test%22%7D"  # pylint: disable=line-too-long
         )
 
     @missing_feature(
-        context.library in ["golang", "nodejs", "php", "ruby"],
+        context.library in ["golang", "nodejs", "php", "ruby", "python", "java"],
         reason="tracer did not yet implemented the new version of query parameters obfuscation regex",
     )
-    @missing_feature(context.library < "java@1.21.0", reason="previous obfuscation regex")
-    @irrelevant(context.library < "python@1.19", reason="python released the new version at 1.19.0")
+    # @missing_feature(context.library < "java@1.21.0", reason="previous obfuscation regex")
+    # @irrelevant(context.library < "python@1.19", reason="python released the new version at 1.19.0")
```

Comment:
If they are no longer needed, it is better to delete

Reply:
yes, fixed in [4430067](https://github.com/DataDog/system-tests/pull/1756/commits/44300678c50738d4409bfac328ab5bb7b43f5363)